lesson_2/Lesson_2.4.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>mass <strong>is</strong> associated with 1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>